app/poros/one_merchant.rb

```diff
@@ -0,0 +1,7 @@
+class OneMerchant
```

Comment:
This class appears to be identical to the `Merchants` poro. I don't think you need both. You should be able to replace `OneMerchant.new` with `Merchants.new` and still have it work.